rules/common/notificationHubTemplate.ts

```diff
@@ -0,0 +1,21 @@
+import { danger, warn } from "danger"
+
+// PRs changing Notification Hub templates should have reminders to update the template on the Notification Hub Adm,in
+const notificationHubTemplate = async () => {
+  const isOpen = danger.github.pr.state === "open"
+
+  if (isOpen) {
+    const files = [...danger.git.modified_files, ...danger.git.created_files]
+    const templates_re = /(templates\/.*)/
+
+    const hasTemplateChanges = files.find(file => templates_re.test(file))
+
+    if (hasTemplateChanges) {
```

Comment:
Looks like "danger" has a nice [API](https://github.com/danger/danger-js/blob/main/source/dsl/GitDSL.ts#L134-L139) for finding file matching a certain globbed string (eg: `danger.git.fileMatch("**/*.md")`).

So, in theory we could simply do here:
```
  if (isOpen) {
    const emailTemplates = danger.git.fileMatch("templates/*.html")
    const hasTemplateChanges = emailTemplates.created || emailTemplates.modified || emailTemplates.edited || emailTemplates.deleted;
    if (hasTemplateChanges) {
```

I included `deleted` state as well, because if people remove the file from this repo they should make sure it is also reflected on Admin as well, right?

Reply:
Right now, in general we don't delete the templates from the Notification Hub Admin actually- we could I suppose but we don't because it's not really necessary

Will  `danger.git.fileMatch("templates/*.html")` also grab files that are structured like `"templates/emails/*.html"` or no?

Reply:
@alexantoniomoran ack on your note regarding deleted files. I wouldn't use the same assumption as yours, but I get your point.

> Will danger.git.fileMatch("templates/*.html") also grab files that are structured like "templates/emails/*.html" or no?

We might need to add something like "templates/**/*.html" if we want to include nested folder structures. We need to double check the documentation of the appropriate globbed pattern here. 

I just realized that the PR is mostly approved, so please feel free to judge if this minor suggestion here is beneficial or not. Nothing is extremely critical from my end, so feel free to move ahead.